src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/Interfaces/IProxyDataCollectionManager.cs

```diff
@@ -36,6 +39,7 @@ public interface IProxyDataCollectionManager : IDisposable
         /// BeforeTestRunStartResult object
         /// </returns>
         DataCollectionParameters BeforeTestRunStart(
+            TestRunCriteria testRunCriteria,
```

Comment:
No need to change this interface as testRunCriteria will come from constructor

Reply:
Done